.Gruntfile.babel.js

```diff
@@ -483,10 +437,6 @@ function svgLoader (options) {
       const svgName = path.basename(filePath, '.svg')
       const svgClass = `svg-${svgName}`
       const svg = source
-        // remove width and height attrs to be customized only by CSS
-        // because SVGO can't do it at the moment (bug): https://github.com/svg/svgo/issues/871
-        .replace(/width="[a-zA-Z0-9:]*"/, '')
-        .replace(/height="[a-zA-Z0-9:]*"/, '')
```

Comment:
Note: As I explained at https://github.com/svg/svgo/issues/871#issuecomment-533801800, it seems this was a bug from svg-sprite, not SVGO itself. So we don't need to do it manually anymore.